README.md

```diff
@@ -22,7 +22,6 @@ const permanent = new Permanent({
   sessionToken,
   mfaToken,
   archiveNbr,
-  apiKey,
```

Comment:
Should we update the text description above this code snippet? It still mentions access keys which must be requested from us, and I *think* that's referring to the API key?

Reply:
Yeah, I looked at that, and punted; you're right that it needs to be cleaned up. I think at this point that's referring to authentication instructions, so users do still need to talk to us, but I can make that clearer. Thanks!

Reply:
New language:

> The client needs to be configured with authentication tokens. Please contact engineers@permanent.org for access.